scripts/mobility-database-harvester/harvest_latest_versions.py

```diff
@@ -11,79 +11,28 @@
 # limitations under the License.
 
 import argparse
-
+import json
+import pandas as pd
 import numpy as np
-import requests
-from datetime import datetime as dt
 import os
 from os import path
-import json
-from tqdm import tqdm
 
-###############################################################################
-# This script harvests the latest dataset versions on the Mobility Database.
+#####################################################################################
+# This script harvests the latest dataset versions on the Mobility Database Catalogs.
 # Made for Python 3.9. Requires the modules listed in requirements.txt.
```

Comment:
I think in general this script could use more documentation to help subsequent developers (and code reviewers like me) understand the behavior, the inputs, and the outputs of this script without having to dig through so much code.